dao/src/main/java/com/epam/esm/dao/util/QueryCreator.java

```diff
@@ -0,0 +1,51 @@
+package com.epam.esm.dao.util;
+
+import java.util.Map;
+
+public class QueryCreator {
+  private final static String WHERE_RE = ".*WHERE.*";
+  private final static String WHERE = "WHERE ";
+
+  private final static String ORDER_RE = ".*ORDER.*";
+  private final static String ORDER_BY = "ORDER BY ";
+  private final static String SORT = "DESC";
+
+  private final static String SORT_RE = "^SORT_.*";
+  private final static String SEARCH_RE = "^SEARCH_.*";
```

Comment:
please rename such variables with _RE in the end. It takes some time to realize that this means regular expression

Reply:
Ok